src/rqg.scss

```diff
@@ -272,6 +276,20 @@
       margin-right: -0.4rem;
     }
 
+    .footer {
+      grid-column: 1/-1;
+      grid-row: auto;
+      margin-top: -1.3rem;
+      background: var(--rqg-color-main-bg);
+      mix-blend-mode: color-burn;
+      text-shadow: 3px 4px 4px var(--rqg-color-grid-text-shadow);
+      box-shadow: var(--rqg-box-shadow-grid-header);
+      border: 3px double grey;
```

Comment:
Could we meet halfway :-) What I'm after is differentiating headers and footer, but this is a little less differentiating than my previous suggestion: 
```suggestion
      border: 1px solid grey;
```
 
![image](https://user-images.githubusercontent.com/4853677/147481435-6507e3a1-960b-4a98-8354-852aee1f04da.png)

Reply:
> Could we meet halfway :-) What I'm after is differentiating headers and footer, but this is a little less differentiating than my previous suggestion:

I'm no visual artist but that's the best solution so far, I think.  Looks similar but is obviously different.